table_foosball/text_tools.rb

```diff
@@ -0,0 +1,51 @@
+module TextTools
+	def find_matching_bracket(strings, index)
+		bracket_count = 1
+
+		for j in index+1...strings.count
+			if strings[j].last == "["
+				bracket_count += 1
+			elsif strings[j].last == "]"
+				bracket_count -= 1
+			end
+			return j if bracket_count == 0
+		end
+
+	end
+
+	def find_next_object(strings, index)
```

Comment:
Why did you decide to put these in a different module? They belong perfectly inside of the `Database` class as private methods. Usually you'd use a mixin if it's reuseable and can be tested independently, but in this case you're doing neither.